code/game/objects/items/weapons/storage/nitrilebox.dm

```diff
@@ -1,78 +0,0 @@
-/obj/item/nitrilebox
-	name = "nitrile gloves box"
-	desc = "Contains nitrile gloves."
-	icon = 'icons/obj/storage.dmi'
-	icon_state = "nitrilebox"
-	item_state = "sheet-metal"
-	drop_sound = 'sound/items/drop/cardboardbox.ogg'
-	pickup_sound = 'sound/items/pickup/cardboardbox.ogg'
-	throwforce = 1
-	w_class = ITEMSIZE_NORMAL
-	throw_speed = 3
-	throw_range = 7
-	layer = OBJ_LAYER - 0.1
-	var/amount = 50					//How much paper is in the bin.  Gloves, actually, but this code is copied from the paper bin code.
-
-/obj/item/nitrilebox/MouseDrop(mob/user)
-	if((user == usr && (!( usr.restrained() ) && (!( usr.stat ) && (usr.contents.Find(src) || in_range(src, usr))))))
-		if(!istype(usr, /mob/living/carbon/slime) && !istype(usr, /mob/living/simple_animal))
-			if( !usr.get_active_hand() )		//if active hand is empty
-				var/mob/living/carbon/human/H = user
-				var/obj/item/organ/external/temp = H.organs_by_name[BP_R_HAND]
-
-				if (H.hand)
-					temp = H.organs_by_name[BP_L_HAND]
-				if(temp && !temp.is_usable())
-					to_chat(user, SPAN_NOTICE("You try to move your [temp.name], but cannot!"))
-					return
-
-				to_chat(user, SPAN_NOTICE("You pick up [src]."))
-				user.put_in_hands(src)
-
-	return
-
-/obj/item/nitrilebox/attack_hand(mob/user)
-	if(ishuman(user))
-		var/mob/living/carbon/human/H = user
-		var/obj/item/organ/external/temp = H.organs_by_name[BP_R_HAND]
-		if (H.hand)
-			temp = H.organs_by_name[BP_L_HAND]
-		if(temp && !temp.is_usable())
-			to_chat(user, SPAN_NOTICE("You try to move your [temp.name], but cannot!"))
-			return
-	if(amount > 0)
-		var/list/glove_types = list(
-			"Standard Gloves" = /obj/item/clothing/gloves/latex/nitrile,
-			"Unathi Gloves" = /obj/item/clothing/gloves/latex/nitrile/unathi,
-			"Tajara Gloves" = /obj/item/clothing/gloves/latex/nitrile/tajara,
-			"Vaurca Gloves" = /obj/item/clothing/gloves/latex/nitrile/vaurca
```

Comment:
The Nitrile Glove box is a separate item precisely because it gives the option for the various species specific gloves. Removing this straight up is really bad, dude.

Reply:
kind of defeats the purpose of being a unique species if it boils down to just clicking a different button when picking out your nitrile gloves, so i think it will add to the immersion.

speaking of which, i've made snipping gloves more loose by using issharp() proc instead, so it's more easy.
I also put them in their respective Xenowear loadouts if people ever need to access them.

Reply:
alright. i've put xenowear gloves back in the box, and removed them from loadouts.

they should have the same effect as the box without needing an entire page of code for this singular purpose.